stor/dx.py

```diff
@@ -176,22 +178,59 @@ def joinpath(self, *others):
         """
         return self.path_class(self.path_module.join(self.normpath(), *others))
 
+    def to_url(self):
+        """For compatibility with OBS - returns ``temp_url()``"""
+        return self.temp_url()
+
     def temp_url(self, lifetime=300, filename=None):
         """Obtains a temporary URL to a DNAnexus data-object.
 
+        If ``DX_FILE_PROXY_URL`` or ``[dx] file_proxy_url=`` is set, will use that to construct a
+        path instead, e.g.::
+
+            >>> stor.Path('dx://proj:/folder/mypath.csv').temp_url()
+            'https://dl.dnanex.us/F/D/awe1323/mypath.csv'
+            >>> with stor.settings.use({'dx': {'file_proxy_url':
+            ...     'https://myproxy.example.com/gateway'}):
+            ... stor.Path('dx://proj:/folder/mypath.csv').temp_url()
+            'https://my-dnax-proxy.example.com/gateway/proj/folder/mypath.csv'
```

Comment:
```suggestion
            'https://myproxy.example.com/gateway/proj/folder/mypath.csv'
```

Reply:
good catch, I'm going to change the one above tho :)